src/main/java/executor/service/utils/WebDriverInitializer.java

```diff
@@ -0,0 +1,68 @@
+package executor.service.utils;
+
+import executor.service.model.ProxyConfigHolder;
+import executor.service.model.ProxyCredentials;
+import executor.service.model.ProxyNetworkConfig;
+import executor.service.model.WebDriverConfig;
+import org.openqa.selenium.Proxy;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import java.time.Duration;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class WebDriverInitializer {
+    private static final Logger LOGGER = Logger.getLogger(WebDriverInitializer.class.getName());
+
+    public WebDriver init(WebDriverConfig webDriverConfig, ProxyConfigHolder proxyConfigHolder) {
```

Comment:
where do we get WebDriverConfig? we need to parse it from properties. And ProxyConfigHolder comes from ProxySourcesClient.getProxy(). Correct me, if I am mistaken.

Reply:
Or should DI factory parse configs?